system/jlib/jstring.cpp

```diff
@@ -216,13 +216,15 @@ StringBuffer & StringBuffer::append(const char * value)
     return *this;
 }
 
-StringBuffer & StringBuffer::append(unsigned len, const char * value)
+StringBuffer & StringBuffer::append(size_t len, const char * value)
```

Comment:
Slightly concerned about what the impact is. Any case that is currently going wrong because of this is going to still fail afterwards (just differently). Would have to be trying to create a string >4Gb to be impacted too...